src/main/resources/swagger/api-docs.yaml

```diff
@@ -2720,9 +2720,20 @@ definitions:
 
   SearchTermRef:
     properties:
-      searchTerm:
+      searchString:
         type: string
         description: The term to search for
+      searchFields:
+        type: object
+        description: Map[String, Array[String]] Each entry contains the column name (i.e. "library:indication") and a list of terms that will be or'ed in the query (["cancer", "sleep apnea"])
```

Comment:
searchString is the free text search
searchFields will contain the facet selections - do you prefer a different name? matchFields? matchTerms?